File: 226_Invert_Binary_Tree.java
class Solution {
    public TreeNode invertTree(TreeNode root) {

        if(root == null){
            return root;
        }

        TreeNode left = root.left;
        TreeNode right = root.right;

        root.right = left;
        root.left = right;

        invertTree(root.left);
        invertTree(root.right); 
        
        return root;
        
    }
}

File: 226. Invert BInary Tree.py
class Solution:
    def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
        if not root:
            return
        
        temp = root.right
        root.right = root.left
        root.left = temp

        self.invertTree(root.left)
        self.invertTree(root.right)
        
        return root
